### ISSUE TITLE
Various features not working

### ISSUE THREAD

**Issue by PancakeBoy27** [Pandarix/BetterArcheology#93]
I've been using the mod for a while, but yesterday I wanted to play with more features, but found out that Totem of Growth can't be picked up, nor broken by any tool so it drops, also placing a block next to it breaks it. Besides this, I found out I can't get any items from ancient vases, can't craft the archeology table, and can't craft artifacts from ancient shards. Totems of Radiance also can't be broken so they drop. 

Os this a known issue? Might be a mod that's causing it? Please help!

**Comment by Pandarix** [comment]
I am very sorry you are experiencing these issues. What ModLoader and version of BetterArcheology are you using?

**Comment by PancakeBoy27** [comment]
Minecraft 1.20.1, Fabric 0.15.11 on CurseForge, Better Archeology-1.2.1! Thanks for the quick response

**Comment by Pandarix** [comment]
Thanks!
I will try to replicate this issue and get back to you once I've found out more or need more information.
Np that's my job. I hate it when people have a bad time using my mods ^^'

**Comment by Pandarix** [comment]
I could replicate all of your problems. My suspicion is that somewhere along the lines, I did not update the generated Data. All of the problems you describe could be traced back to the .json files e.g. determining the crafting recipes or loot tables.
This issue seems not to be in the forge version.
I'll keep you updated

**Comment by Pandarix** [comment]
TLDR: I was right with my suspicion. Apparently all of the datagenerated json files were malformed or corrupted.

> [!NOTE] 
> When I regenerated the files, there were absolutely no differences displayed. However, they seem to have had differences in line seperators.
After trying to commit these changes, the push failed because there were no changes to commit. So I don't have a fix commit to show for it, but I will be uploading the fixed jar asap.

The old version will get archived then. I'll send a link to the new file, once it is up :).

Thank you for your quick response with more information as I'm quite caught up with the update to 1.21 ^^'

**Comment by PancakeBoy27** [comment]
Thanks! 10/10 costumer service! :D Looking forward to playing the mod

**Comment by Pandarix** [comment]
Thanks ^^! [CurseForge seems to have issues right now](https://downforeveryoneorjustme.com/curseforge). I'll do what I can.
Hope you'll enjoy the mod :)

**Comment by Pandarix** [comment]
## Downloads
[![Modrinth Downloads](https://img.shields.io/modrinth/dt/better-archeology?style=flat&logo=modrinth&link=https%3A%2F%2Fmodrinth.com%2Fmod%2Fbetter-archeology%2Fversions)](https://modrinth.com/mod/better-archeology/version/QNQXrdhi)
> [!NOTE]
> I will add a CurseForge Download to this comment once their servers are usable again.

Please tell me if that fixed your problems and if so, feel free to close this issue :)

**Comment by PancakeBoy27** [comment]
It works! Thanks for the fix! I'll close the issue right away.
